src/notification_manager/mod.rs

```diff
@@ -490,6 +530,15 @@ impl NotificationManager {
         {
             return Ok(false);
         }
+        if notification_preferences.hellthread_notifications_disabled
+            && self.is_hellthread_eligible(event.kind())
+        {
+            if let Ok(pubkeys_count) = self.pubkeys_referenced_by_event(event).len() {
```

Comment:
I am hitting a compiler error in this line:

```
error[E0308]: mismatched types
   --> src/notification_manager/mod.rs:536:20
    |
536 |             if let Ok(pubkeys_count) = self.pubkeys_referenced_by_event(event).len() {
    |                    ^^^^^^^^^^^^^^^^^   --------------------------------------------- this expression has type `usize`
    |                    |
    |                    expected `usize`, found `Result<_, _>`
    |
    = note: expected type `usize`
               found enum `std::result::Result<_, _>`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `notepush` (lib) due to 1 previous error
```

Maybe we need to do some conversion. Here is one suggestion:
```suggestion
            if let Ok(pubkeys_count) = i8::try_from(self.pubkeys_referenced_by_event(event).len()) {
```

Reply:
Sorry, I missed that. Fixed!